### DISCUSSION TITLE
要望・不具合[qiita-cli]: `pnpx qiita init` が実行できない

### DISCUSSION THREAD

**Discussion by ghost** [increments/qiita-discussions#602] [要望・アイデア・不具合]
[qiita-cliのガイドにあるコマンド](https://github.com/increments/qiita-cli?tab=readme-ov-file#2-qiita-cli-%E3%82%92%E3%82%A4%E3%83%B3%E3%82%B9%E3%83%88%E3%83%BC%E3%83%AB%E3%81%99%E3%82%8B)をpnpmで実行した時に、`ERR_PNPM_DLX_NO_BIN`エラーが発生します。

## 実行したコマンド
```
pnpx qiita init
```

## 実行結果
[![asciicast](https://asciinema.org/a/630115.svg)](https://asciinema.org/a/630115)

## 考えられる原因
npm registry に[`qiita`](https://www.npmjs.com/package/qiita)という名前の他のパッケージがあるから。

## 解決策
```
pnpx @qiita/qiita-cli init
```

---

素晴らしいツールを開発していただき本当にありがとうございます。
npm以外のパッケージマネージャーを利用しているQiitaユーザーも増えてきていると思うので、対応していただけると嬉しいです。

**Comment by ohakutsu** [comment]
@nabeken5 
ご報告ありがとうございます！

pnpmの場合は、`npx`の部分を`pnpm exec`に読み替えていただけると動作するようになります！

```
$ pnpm install -D @qiita/qiita-cli
$ pnpm exec qiita version
1.3.1

$ pnpm exec qiita init
```

また、[README](https://github.com/increments/qiita-cli/blob/main/README.md) にも記載させていただきます！
